lib/blacklight/document.rb

```diff
@@ -15,6 +15,7 @@
 module Blacklight::Document
   autoload :ActiveModelShim, 'blacklight/document/active_model_shim'
   autoload :SchemaOrg, 'blacklight/document/schema_org'
+  autoload :CacheKey, 'blacklight/document/cache_key'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.